server/utils.py

```diff
@@ -30,6 +31,10 @@
 TWENTY_FOUR_HOURS = 86400
 
 
+def db_table_exists(table_name):
+    return table_name in connection.introspection.table_names()
```

Comment:
This is the issue. This is always resolving as `False` which allows it to pass the test during migrations.